### ISSUE TITLE
Slider Tooltip overflow hidden

### ISSUE THREAD

**Issue by Risky-Shot** [FeatherFramework/feather-menu#9]
Hi,

Tooltip of slider element is half cut when placed as first element in 'content' slot.

![image](https://github.com/FeatherFramework/feather-menu/assets/52458646/23c8695f-f18a-46eb-874e-cc6bc9f58d2f)

Code to Reproduce : 

```
local oilWellCollectMenu = FeatherMenu:RegisterMenu('oilFactory:oilWell:Collect', {
        top = '1%',
        left = '1%',
        ['720width'] = '500px',
        ['1080width'] = '600px',
        ['2kwidth'] = '700px',
        ['4kwidth'] = '900px',
        style = { },
        contentslot = {
            style = { --This style is what is currently making the content slot scoped and scrollable. If you delete this, it will make the content height dynamic to its inner content.
                ['height'] = '300px',
                ['min-height'] = '300px'
            }
        },
        draggable = true,
        --canclose = false
    }, {
        opened = function()
            print("MENU OPENED!")
        end,
        closed = function()
            print("MENU CLOSED!")
        end,
        topage = function(data)
            print("PAGE CHANGED ", data.pageid)
        end
    })

    local oilWellCollectPage = oilWellCollectMenu:RegisterPage('first:page')
    oilWellCollectPage:RegisterElement('header', {
        value = 'Oil Factory',
        slot = "header",
        style = {
            ['color'] = 'rgb(127, 29, 29)'
        }
    })
    oilWellCollectPage:RegisterElement('subheader', {
        value = "Collect Crude Oil",
        slot = "header",
        style = {}
    })

    oilWellCollectPage:RegisterElement('line', {
        slot = "header",
        style = {}
    })

    oilWellCollectPage:RegisterElement('slider', {
        label = "Amount",
        slot = "content",
        start = 1,
        min = 1,
        max = 10,
        steps = 1,
        style = { }
    }, function(data)
        -- This gets triggered whenever the sliders selected value changes
        print(data.value)
    end)

    oilWellCollectPage:RegisterElement('button', {
        label = "Fill Barrel",
        style = { },
    }, function()
        -- This gets triggered whenever the button is clicked
    end)

    oilWellCollectMenu:Open({
        startupPage = oilWellCollectPage
    })
```

Possible solution : https://nightcatsama.github.io/vue-slider-component/#/api/props?hash=contained

**Comment by AndrewR3K** [comment]
This will be fixed in the next update. Thanks for reporting this. And my apologies for the delay.

Sneak Peak: 
![image](https://github.com/user-attachments/assets/baf06c6d-6165-44c0-b4de-738fb5983444)

**Comment by AndrewR3K** [comment]
Fixed in [1.0.10](https://github.com/FeatherFramework/feather-menu/releases/tag/1.0.10)
